parser/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -2,53 +2,58 @@ package hmda.parser.fi.lar
 
 import hmda.model.fi.lar._
 
-import scala.util.Try
+import scala.collection.immutable.ListMap
+import scalaz._
+import scalaz.Scalaz._
+import scala.util.{ Failure, Success, Try }
 
 object LarCsvParser {
   def apply(s: String): Either[List[String], LoanApplicationRegister] = {
-    val larErrors = checkLar(s)
+    val values = s.split('|').map(_.trim)
+    val larErrors = checkLar(values.toList)
+    val validation = larErrors.disjunction
```

Comment:
Bit of a nitpick, but validation is overloaded in our code base.  Maybe rename to something like `parseResult`?